third_party/BUILD

```diff
@@ -465,17 +465,13 @@ filegroup(
     ],
 )
 
-distrib_java_import(
+java_library(
     name = "guava",
-    enable_distributions = ["debian"],
-    jars = [
-        "guava/failureaccess-1.0.1.jar",
-        "guava/guava-31.1-jre.jar",
-    ],
     exports = [
-        ":error_prone_annotations",
-        ":jcip_annotations",
-        ":jsr305",
+       "@maven//:com_google_guava_guava",
+       "@maven//:com_google_errorprone_error_prone_annotations",
```

Comment:
I am a little confused by this. You still have //third_party:error_prone_annotations. That is used all over the place. Is this even needed in the export? Or should this java library depend on one of the two error_prone_annotations?

Reply:
Nice catch! I was essentially duplicating error_prone_annotations here. I fixed this by replacing all jars in the transitive closure of guava so that no jars are checked-in and fetched by rules_jvm_external at the same time.